server/src/promo-dialog.js

```diff
@@ -0,0 +1,51 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("./browser-send-event.js");
+
+exports.PromoDialog = class PromoDialog extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (this.props.display) {
+      this.props.promoOpen(this.props.promoType);
+      return <div id="promo-dialog-panel" className="promo-panel" >
+        <div className={`${this.props.promoType} promo-box default-color-scheme`}>
+        <a className="box-close" onClick={this.closePanel.bind(this)}></a>
+        <Localized id="promoTitle">
+          <span className="title">Take Note!</span>
+        </Localized>
+        <Localized id="promoMessage">
+          <p className="message">
+            Updated editing tools let you crop, highlight, and even add text to your shot.
+          </p>
+        </Localized>
+        <span className="promo-star"></span>
+         <Localized id="promoLink">
+          <span id="promo-link" className="message link" onClick={this.closePanel.bind(this)}>
```

Comment:
But this probably should be a link or button, since it is interactable. You could use `aria-role`, but that's not great, while just making it an `<a>` or `<button>` basically solves the accessibility issue.

Reply:
Updated to <a> and use aria-label as "Try Edit Link"